### ISSUE TITLE
Steam is unable to open the Controller Configurate on the desktop client

### ISSUE THREAD

**Issue by Managor** [ValveSoftware/steam-for-linux#6843]
#### Your system information

* Steam client version: 
![image](https://user-images.githubusercontent.com/42655600/73015114-204ec780-3e24-11ea-9ae0-ddef53774035.png)

* Distribution: Arch
* Opted into Steam client beta?: Yes
* Have you checked for system updates?: Yes

#### Please describe your issue in as much detail as possible:
When having the desktop client open, right clicking on a game and selecting the Controller Configurator does nothing.
[Demonstrative video here.](https://github.com/ValveSoftware/steam-for-linux/files/4104954/Peek.2020-01-23.17-32.zip)

#### Steps for reproducing this issue:

1. Opt into the beta
2. Try to open the Controller Configurator on desktop mode

**Comment by Smeejo** [comment]
I'm having the same issue, am on solus. Tried every game I have installed, tried right clicking and selecting controller configuration, left clicking the game and clicking controller configuration on the game page itself. 

I've tried closing steam and relaunching it. Tried restarting my computer. Checked for all updates.

Steam info:

> built: Feb 11 2020, at 19:27:00
steam api: v020
steam package version: 1581460722

My inxi -F

> Kernel: 5.4.12-144.current x86_64 bits: 64 
  Desktop: Budgie 10.5.1 Distro: Solus 4.1 
Machine:
  Type: Desktop Mobo: ASUSTeK model: ROG STRIX X470-F GAMING v: Rev X.0x 
  serial: <root required> UEFI: American Megatrends v: 5220 date: 09/12/2019 
CPU:
  Topology: 6-Core model: AMD Ryzen 5 2600X bits: 64 type: MT MCP 
  L2 cache: 3072 KiB 
  Speed: 2690 MHz min/max: 2200/3600 MHz Core speeds (MHz): 1: 3392 2: 2405 
  3: 2878 4: 2599 5: 2407 6: 2966 7: 3013 8: 2464 9: 2540 10: 2328 11: 2649 
  12: 4135 
Graphics:
  Device-1: NVIDIA GM107 [GeForce GTX 750 Ti] driver: nvidia v: 440.44 
  Display: x11 server: X.Org 1.20.6 driver: nvidia 
  resolution: 1920x1080~75Hz 
  OpenGL: renderer: GeForce GTX 750 Ti/PCIe/SSE2 v: 4.6.0 NVIDIA 440.44 
Audio:
  Device-1: NVIDIA driver: snd_hda_intel 
  Device-2: AMD Family 17h HD Audio driver: snd_hda_intel 
  Sound Server: ALSA v: k5.4.12-144.current 
Network:
  Device-1: Intel I211 Gigabit Network driver: igb 
  IF: enp6s0 state: up speed: 100 Mbps duplex: full mac: 04:d4:c4:57:dc:96 
Drives:
  Local Storage: total: 465.76 GiB used: 92.49 GiB (19.9%) 
  ID-1: /dev/sda vendor: Crucial model: CT500MX500SSD1 size: 465.76 GiB 
Partition:
  ID-1: / size: 453.31 GiB used: 92.38 GiB (20.4%) fs: ext4 dev: /dev/sda3 
  ID-2: /boot size: 487.3 MiB used: 113.2 MiB (23.2%) fs: vfat 
  dev: /dev/sda1 
  ID-3: swap-1 size: 3.73 GiB used: 512 KiB (0.0%) fs: swap dev: /dev/sda2 
Sensors:
  System Temperatures: cpu: 55.1 C mobo: N/A gpu: nvidia temp: 31 C 
  Fan Speeds (RPM): N/A gpu: nvidia fan: 40% 
Info:
  Processes: 275 Uptime: 3d 4h 50m Memory: 15.61 GiB used: 2.30 GiB (14.7%) 
  Shell: bash inxi: 3.0.37
